src/index.html

```diff
@@ -3,11 +3,396 @@
+                    class="events__image3"
+                    alt="walt Disney"
+                  >
+                  
+                </div>
+  
+              </div>
+  
+              
+            
+              <a href="#" class="events__caption-link">
+                Inspiring Walt Disney: 
+                The Animation of French Decorative Arts
+              </a>
+              <p class="events__caption">
+                December 10, 2021 - March 6, 2022
+              </p>
+                
+                
+            </section>
+
+          </section>
+          
+        </div>
+
+        <button class="events__exhibition">View all exhibitions
+
+        </button>
+       
+
+        
+        <section class="explore">
+          <h2 class="explore__title">More to Explore Online</h2>
+          <div class="explore__conntainer">
+            <div class="explore__content">
+              <img src="images/explore1.png" alt="image" class="explore__image">
+              <a href="#" class="explore__link">Collection Areas</a>
+              <p class="explore__intro">These collection areas study, exhibit, and care for the objects in The Met collection.</p>
+            </div>
+  
+            <div class="explore__content">
+              <img src="images/explore2.png" alt="image" class="explore__image">
+              <a href="#" class="explore__link">Travel</a>
+              <p class="explore__intro">Travel with The Met connects the Museum's wide and diverse audience to art, inspiration, and historical perspective.</p>
+            </div>
+            <div class="explore__content">
+              <img src="images/explore3.png" alt="image" class="explore__image">
+              <a href="#" class="explore__link">Virtual group tours</a>
+              <p class="explore__intro">Travel with The Met connects the Museum's wide and diverse audience to art, inspiration, and historical perspective.</p>
+            </div>
+
+          </div>
+          
+
+        </section>
+
+        <section class="contact">
+          <div class="contact-container">
+
+            <h2 class="contact__title">Contact us</h2>
+
+            <h2 class="contact__name">The Met Fifth Avenue</h2>
+            <h2 class="contact__phone">Phone</h2>
+            <h2 class="contact__phoneNumber">+1 212-535-7710</h2>
+            <h2 class="contact__address">address</h2>
+            <h2 class="contact__addressNumber">1000 Fifth Avenue
+              New York, NY 10028
+            </h2>
+            <h2 class="contact__name">The Met Cloisters</h2>
+            <h2 class="contact__phone">Phone</h2>
+            <h2 class="contact__phoneNumber">+1 212-923-3700</h2>
+            <h2 class="contact__address">address</h2>
+            <h2 class="contact__addressNumber">
+              99 Margaret Corbin Drive
+              Fort Tryon Park
+              New York, NY 10040
+            </h2>
+
+          </div>
+
+          <div class="info">
```

Comment:
not div, it should be a `<form`
>13	Placeholders in the forms suggest what to enter; apply validation of the form fields (required, email / tel etc.), then it is clear in what format to enter the data
>14	Form shouldn't be submitted if some of the fields are not filled
>15	"Page shouldn't be reloaded on form submit (`<form onsubmit=""event.preventDefault()""`)
(OR https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault)"